pennylane_lightning/core/src/simulators/lightning_gpu/measurements/MeasurementsGPU.hpp

```diff
@@ -165,6 +165,32 @@ class Measurements final
         return BaseType::probs(obs);
     }
 
+    /**
+     * @brief Probabilities with shot-noise.
+     *
+     * @param num_shots Number of shots.
+     *
+     * @return Floating point std::vector with probabilities.
+     */
+    std::vector<PrecisionT> probs(const size_t &num_shots) {
+        return BaseType::probs(num_shots);
```

Comment:
It's really nice to see the base class impl can support the measurement process on all devices even on LGPU :100: